lib/Runtime/Language/AsmJsUtils.cpp

```diff
@@ -187,6 +187,21 @@ namespace Js
         }
         return (int64)_wcstoui64(buf, nullptr, radix);
     }
+
+    Var CreateI64ReturnObject(int64 val, ScriptContext* scriptContext)
+    {
+        Js::Var i64Object = JavascriptOperators::NewJavascriptObjectNoArg(scriptContext);
+        Var low = JavascriptNumber::ToVar((uint)val, scriptContext);
+        Var high = JavascriptNumber::ToVar(val >> 32, scriptContext);
+
+        PropertyRecord const * lowPropRecord = nullptr;
+        PropertyRecord const * highPropRecord = nullptr;
+        scriptContext->GetOrAddPropertyRecord(_u("low"), (int)wcslen(_u("low")), &lowPropRecord);
+        scriptContext->GetOrAddPropertyRecord(_u("high"), (int)wcslen(_u("high")), &highPropRecord);
```

Comment:
Yes, I moved it up to place it in `#if ENABLE_DEBUG_CONFIG_OPTIONS` since it's a testing function and should not be used otherwise